run-local/docker-compose.yaml

```diff
@@ -104,7 +104,7 @@ services:
   postgres:
     container_name: "workshop-postgres"
     networks: [ workshop-network ]
-    image: postgres:14-alpine
+    image: postgres:17.2
```

Comment:
let's keep alpine version - it's smaller and quicker to download

Reply:
so change to 17-alpine please, there's also `local-postgres.sh` scipt - update there too